packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -139,7 +147,13 @@ class CodePushClient {
     final response = await _httpClient.send(request);
     final body = await response.stream.bytesToString();
 
-    if (response.statusCode != HttpStatus.ok) throw _parseErrorResponse(body);
+    if (response.statusCode == HttpStatus.conflict) {
+      throw ResourceConflictException(
```

Comment:
Thoughts on not overwriting the message from the server? We're duplicating the same message in multiple places and the CLI isn't using the message for anything at the moment so it seems unnecessary.